app/src/main/java/com/codesoom/assignment/Constant.java

```diff
@@ -0,0 +1,17 @@
+package com.codesoom.assignment;
+
+/**
+ * Class have constants used in application
+ *
+ * @author Taeheon Woo
+ * @version 1.0
+ *
+ */
```

Comment:
- 클래스를 설명하는 주석에 `Class`라고 적을 필요는 없습니다.
- git이 있으므로 실용적인 이유로 `@author`는 생략할 수 있습니다. 
- `@version`도 지금은 생략하는 것이 좋아 보입니다.

다음과 같이 이 클래스가 하는 일을 단순하게 설명해주는 것으로 충분합니다.
```suggestion
/**
 * Http 프로토콜 처리와 관련된 상수 정의를 제공합니다.
 */
```